content/technotes/2019-01-10-hugo.md

```diff
@@ -0,0 +1,109 @@
+
+### All the feeds
+
+Our website has both RSS and JSON feeds. We added a [JSON feed](https://jsonfeed.org/), feed format introduced in May last year, because we could and because some people might like that. 
+
+* RSS feeds https://ropensci.org/blog/index.xml, https://ropensci.org/technotes/index.xml
+
+* JSON feeds https://ropensci.org/blog/index.json, https://ropensci.org/technotes/index.json
+
+Here's [how we added the JSON feeds](https://github.com/ropensci/roweb2/pull/315).
+
+### Not too much content for R-Bloggers
+
+R-Bloggers feed https://ropensci.org/rbloggers/index.xml
+
+We created an [R-Bloggers page which only outputs XML](https://github.com/ropensci/roweb2/blob/master/content/rbloggers/_index.md), with [an XML template](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/rbloggers/rss.xml) ensuring this RSS feed is a full RSS feed contrary to the rest of our website that uses Hugo default RSS feeds with excerpts. We do not filter posts by category which means this not-so-R post (but still a bit R, thanks to our explaining how to browse our R stuff ;o) will appear in R-Bloggers, but we might do that later on.
+
+## Twitter friendliness
+
+While Twitter is not the only platform on which our content is shared, it is an important player so we've upped our Twitter game with two edits.
+
+{{< tweet 1073519094958313472>}}
+
+We've worked on adding Twitter metadata to our [head partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/head.html) so when tweeting links from the website you get a pretty card (see above). We've used [resources listed in this GitHub issue](https://github.com/ropensci/roweb2/issues/290#issue-367542204) and [Twitter card validator](https://cards-dev.twitter.com/validator) on Netlify preview links. Whilst updating the head partial we realized not all our pages used it so we homogeneized all pages. Some of our pages like [about](/about) have fancy stuff that's not Markdown based, but we now handle that with partials listed in YAML and then called from the [single.html default template](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/_default/single.html). Here [the source of our about page](https://github.com/ropensci/roweb2/blob/master/content/about.md).
+
+{{< tweet 953090299819118592 >}}
+
+We _put a bird on it_, in Mara Averick's words, i.e. we now add a Twitter icon near authors' names when their metadata contains a Twitter username (have you noticed the bird near my name at the top of this post?). That happens [in the authors.html partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/authors.html). Authors pages also link to Twitter accounts. By the way if you're ever authored a post for us feel free to open a pull request updating your metadata, e.g. if you were Ignasi Bartomeus you'd update [this file](https://github.com/ropensci/roweb2/blob/master/content/authors/ignasi-bartomeus/_index.md).
```

Comment:
```suggestion
We _put a bird on it_, in Mara Averick's words, i.e. we now add a Twitter icon near authors' names when their metadata contains a Twitter username (have you noticed the bird near my name at the top of this post?). That happens [in the authors.html partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/authors.html). Authors pages also link to Twitter accounts. By the way, if you're ever authored a post for us feel free to open a pull request updating your metadata, e.g. if you were Ignasi Bartomeus you'd update [this file](https://github.com/ropensci/roweb2/blob/master/content/authors/ignasi-bartomeus/_index.md).
```

Reply:
I'm not sure why the diff sometimes doesn't show suggested changes for commit. let me know if in future I should note what suggested changes are made

Reply:
it's fine, _I_ should remember to use all your suggestions before making any other change, because now they are outdated and I need to add them manually :woman_facepalming: